cibuildwheel/windows.py

```diff
@@ -73,10 +74,7 @@ def get_python_configurations(
 
     python_configurations = [PythonConfiguration(**item) for item in full_python_configs]
 
-    map_arch = {
-        "32": Architecture.x86,
-        "64": Architecture.AMD64,
-    }
+    map_arch = {"32": Architecture.x86, "64": Architecture.AMD64, "ARM64": Architecture.ARM64}
```

Comment:
It occurs to me that we should update the `build_platforms.toml` format to use the same arch format as the rest of the codebase, rather than having this map here. But we don't have to solve that in this PR.